<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIController</class>
 <widget class="QWidget" name="UIController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2137</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QMenuBar" name="menubar"/>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,5">
     <item>
      <widget class="QScrollArea" name="scrollAreaForSystems">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>299</width>
          <height>850</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="systemsSectorNameLabel">
           <property name="text">
            <string>Systems</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="systemsComboBox"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="findeNewSystemsBtn">
             <property name="text">
              <string>Finde new</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="stopSelectedSystemBtn">
             <property name="text">
              <string>Stop</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTextBrowser" name="systemDescriptionTextBrowser"/>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea_2">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaForSensors">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>298</width>
          <height>850</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="sensorsSectorNameLable">
           <property name="text">
            <string>Sensors</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="sensorsComboBox"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPushButton" name="recordSensorToFileBnt">
             <property name="text">
              <string>Write to file</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="settingBtn">
             <property name="text">
              <string>Settings</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTextBrowser" name="sensorDescriptionTextBrowser"/>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaForChart">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1502</width>
          <height>850</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
